corgi/api/filters.py

```diff
@@ -128,6 +128,12 @@ class Meta:
         method="filter_root_components",
         label="Show only root components (source RPMs, index container images)",
     )
+
```

Comment:
order is significant in django filters - this ensures **active_streams** is applied before **latest_components_by_streams** to avoid superfluous processing